### ISSUE TITLE
[Host] Publishing to a request/response queue (no replyto header)

### ISSUE THREAD

**Issue by EtherZa** [zarusz/SlimMessageBus#271]
Publishing an event to a request/response queue results in an exception being thrown because no `ReplyTo` header has been set.

[MessageBusBase.cs](https://github.com/zarusz/SlimMessageBus/blob/master/src/SlimMessageBus.Host/MessageBusBase.cs#L624)
```
if (!requestHeaders.TryGetHeader(ReqRespMessageHeaders.ReplyTo, out object replyTo))
{
    throw new MessageBusException($"The header {ReqRespMessageHeaders.ReplyTo} was missing on the message");
}
```

It could be useful to instead perform a no-op when the header is missing to allow for a single handler to be used by both `IRequestResponseBus.Send` and `IPublishBus.Publish` (where the response can be ignored).

**Comment by zarusz** [comment]
Makes sense to change!

**Comment by zarusz** [comment]
Closed by #272
